ext/settings.html

```diff
@@ -212,6 +212,22 @@ <h1>Yomitan Settings</h1>
                 <label class="toggle"><input type="checkbox" data-setting="general.enable"><span class="toggle-body"><span class="toggle-track"></span><span class="toggle-knob"></span></span></label>
             </div>
         </div></div>
+        <div class="settings-item"><div class="settings-item-inner settings-item-inner-wrappable">
+            <div class="settings-item-left">
+                <div class="settings-item-label">
+                    Language
+                </div>
+                <div class="settings-item-description">
+                    The language you are reading with this profile
```

Comment:
Puncutation for descriptions.

Reword to avoid the word "you". (Almost all of the content on this page is intentionally worded to avoid using 2nd person pronouns unless invoked by user action [or the backup description which I was not around to review].)